legged_sirius_hw:
  loop_frequency: 200
  cycle_time_error_threshold: 0.005
  thread_priority: 95
  power_limit: 4
  contact_threshold: 40
